invenio_rdm_records/resources/args.py

```diff
@@ -19,3 +19,4 @@ class RDMSearchRequestArgsSchema(SearchRequestArgsSchema):
     style = fields.Str()
     locale = fields.Str()
     status = fields.Str()
+    is_admin = fields.Bool()
```

Comment:
I don't understand this param, what does it mean?
Checking the code below, it looks like that it is a way to filter "only mine" -> current_user.
I would go for a name more in this direction.